src/styled/shared-card.ts

```diff
@@ -0,0 +1,90 @@
+import styled from '@emotion/styled';
+import theme from '@mdb/flora/theme';
+import { HorizontalRule, Pill, TypographyScale } from '@mdb/flora';
+import { PillCategory } from '../types/pill-category';
+import { pillColorMap } from './pill-color';
+import Image from 'next/image';
+
+const SharedCardWrapper = styled('div')`
+    padding: 32px;
+    height: max-content;
+    border: 1px solid #e7eeec;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
+    border-radius: ${theme.radii.inc50};
+    // Mobile
+    width: 467px;
+    // Desktop small
+    @media only screen and (min-width: ${theme.sizes.breakpoint.large}) {
+        width: 336px;
+    }
+`;
+
+const ThumbnailImage = styled(Image)`
+    border-radius: 8px;
```

Comment:
You have hard-coded `px` values in a couple of places for `padding`, `shadows`, `margins`, etc what I would do is look in `flora` if they use `tokens` for any of these values so we don't have to hard code them.